<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminpanel</class>
 <widget class="QDialog" name="adminpanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Panel</string>
  </property>
  <widget class="QPushButton" name="pushButton_logout">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>231</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButton_adduser_2">
      <property name="text">
       <string>Add User</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_editremove">
        <property name="text">
         <string>Edit/Remove User</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Manage Repairs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Manage Work orders</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Manage Labs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Report Failure</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>Manage Consumables</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_welcome">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
